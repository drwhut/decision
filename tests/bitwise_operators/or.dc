Start~#1

> false OR false = false
Or(false, false)~#2
Print(#1, #2)~#3

> false OR true = false
Or(false, true)~#4
Print(#3, #4)~#5

> true OR false = false
Or(true, false)~#6
Print(#5, #6)~#7

> true OR true = true
Or(true, true)~#8
Print(#7, #8)~#9

> new in 0.3.0: OR integers!
Or(9429, 7626)~#10
Print(#9, #10)~#11