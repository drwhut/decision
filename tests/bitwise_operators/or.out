false
true
true
true
15839
